.github/workflows/testBuildHybrid.yml

```diff
@@ -0,0 +1,352 @@
+          ANDROID_UPLOAD_KEYSTORE_ALIAS: op://Mobile-Deploy-CI/Repository-Secrets/ANDROID_UPLOAD_KEYSTORE_ALIAS
+          ANDROID_UPLOAD_KEY_PASSWORD: op://Mobile-Deploy-CI/Repository-Secrets/ANDROID_UPLOAD_KEY_PASSWORD
+
+      - name: Get Android native version
+        id: getAndroidVersion
+        run: echo "VERSION_CODE=$(grep -o 'versionCode\s\+[0-9]\+' android/app/build.gradle | awk '{ print $2 }')" >> "$GITHUB_OUTPUT"
+
+      - name: Build Android app
+        run: bundle exec fastlane android build_adhoc_hybrid
+        env:
+          ANDROID_UPLOAD_KEYSTORE_PASSWORD: ${{ steps.load-credentials.outputs.ANDROID_UPLOAD_KEYSTORE_PASSWORD }}
+          ANDROID_UPLOAD_KEYSTORE_ALIAS: ${{ steps.load-credentials.outputs.ANDROID_UPLOAD_KEYSTORE_ALIAS }}
+          ANDROID_UPLOAD_KEY_PASSWORD: ${{ steps.load-credentials.outputs.ANDROID_UPLOAD_KEY_PASSWORD }}
+
+  uploadAndroid:
+    name: Upload Android hybrid app to S3
+    needs: [androidHybrid]
+    runs-on: ubuntu-latest
+    outputs:
+      S3_APK_PATH: ${{ steps.exportS3Path.outputs.S3_APK_PATH }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+
+      - name: Setup Ruby
+        uses: ruby/setup-ruby@v1.190.0
+        with:
+          bundler-cache: true
+
+      - name: Download Android build artifacts
+        uses: actions/download-artifact@v4
+        with:
+          path: /tmp/artifacts
+          pattern: android-*-artifact
+          merge-multiple: true
+
+      - name: Log downloaded artifact paths
+        run: ls -R /tmp/artifacts
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Upload AdHoc build to S3
+        run: bundle exec fastlane android upload_s3
+        env:
+          apkPath: /tmp/artifacts/${{ needs.androidHybrid.outputs.APK_FILE_NAME }}
+          S3_ACCESS_KEY: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          S3_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          S3_BUCKET: ad-hoc-expensify-cash
+          S3_REGION: us-east-1
+
+      - name: Export S3 paths
+        id: exportS3Path
+        run: |
+          # $s3APKPath is set from within the Fastfile, android upload_s3 lane
+          echo "S3_APK_PATH=$s3APKPath" >> "$GITHUB_OUTPUT"
+
+  # iOS:
+  #   name: Build and deploy iOS for testing
+  #   needs: [validateActor, getBranchRef]
+  #   if: ${{ fromJSON(needs.validateActor.outputs.READY_TO_BUILD) }}
+  #   env:
+  #     DEVELOPER_DIR: /Applications/Xcode_15.2.0.app/Contents/Developer
+  #   runs-on: macos-13-xlarge
+  #   steps:
+  #     - name: Checkout
+  #       uses: actions/checkout@v4
+  #       with:
+  #         ref: ${{ github.event.pull_request.head.sha || needs.getBranchRef.outputs.REF }}
```

Comment:
Can we leave this out of the initial PR please

Reply:
Sure!